py_oathtool/otp.py

```diff
@@ -98,41 +97,49 @@ def main():
 
         sys.exit()
 
+    # Update the holdoff value if one was specified in the YAML file
+    if 'holdoff' in otpSecrets:
+        global HOLDOFF
+        HOLDOFF = otpSecrets['holdoff']
+
     # Check the label exists
     if args.label in otpSecrets['otpsecrets']:
         try:
             # Do some holdoff magic if the 30s window is almost up
             holdoffCheck = (30 - (datetime.now().second %30))
             if not args.force and holdoffCheck <= HOLDOFF:
-                print 'Short lived OTP. Holding off for %i seconds...' % (holdoffCheck)
+                print('Short lived OTP. Holding off for %i second%s...' % (holdoffCheck, ('' if holdoffCheck == 1 else 's')))
                 time.sleep(holdoffCheck)
 
             # Get the code
             totp = subprocess.check_output(['oathtool', '-b', '--totp', \
-                otpSecrets['otpsecrets'][args.label]]).rstrip('\n')
+                otpSecrets['otpsecrets'][args.label]]).rstrip(b'\n')
```

Comment:
Is adding `b'\n'` the best way to do this in Python 3? It means having to use `decode()` when printing below, but otherwise it results in a TypeError.